.travis.yml

```diff
@@ -12,14 +12,15 @@ jobs:
         # Try to keep version in sync with `package.json`
         - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.19.1
         - export PATH="$HOME/.yarn/bin:$PATH"
+        - export DEBUG_PRINT_LIMIT=1000000
```

Comment:
What is this change and the next one in this file for?

Reply:
Yeah, just for debugging.  Removed this.